docs/chat-configuration/ui-customization/incoming-message/readmore-article.md

```diff
@@ -36,6 +36,21 @@ val customProvider = ChatUIProvider(context).apply {
         articleUIConfig.verticalMargin = x to y // Pair of top, bottom margin values, in pixels
 
         channelsUIProvider = ... // A QuickOptionUIProvider object
+        
+        // Configure the main background of the Article
+        background = ....
```

Comment:
this should part of the apply on the articleUIConfig not on the articleUIProvider
same for the rest